src/screen/Council/CouncilScreen.tsx

```diff
@@ -132,17 +174,205 @@ function Item({
+  visible: boolean;
+  setVisible: (visible: boolean) => void;
+  candidates: string[];
+  module: string | null;
+};
+
+function CouncilVote({visible, setVisible, candidates, module}: CouncilVoteProps) {
+  const [account, setAccount] = React.useState<string>();
+  const [amount, setAmount] = React.useState<string>('');
+  const [selectedCandidates, setSelectedCandidates] = React.useState<Array<string>>([]);
+  const {data: voterData} = useCouncilVotesOf(account);
+
+  // preselect already voted council members
+  useEffect(() => {
+    if (voterData != null) {
+      setSelectedCandidates(voterData.votes.map((a) => a.toString()).filter((a) => candidates.includes(a)));
+    }
+  }, [voterData, candidates]);
+
+  const {api} = useApi();
+  const startTx = useApiTx();
+  const {networkAccounts} = useAccounts();
+  const formatBalance = useFormatBalance();
+
+  const onCandidateSelect = (accountId: string, isSelected: boolean) => {
+    isSelected
+      ? setSelectedCandidates(selectedCandidates.filter((id) => id !== accountId))
+      : selectedCandidates.length < MAX_VOTES
+      ? setSelectedCandidates([...selectedCandidates, accountId])
+      : noop;
+  };
+
+  const disabled = !amount || !account || selectedCandidates.length === 0;
+
+  const bondValue = useMemo(() => {
+    if (api != null) {
+      const location = api.consts.elections || api.consts.phragmenElection || api.consts.electionsPhragmen;
+      return location?.votingBondBase?.add(location.votingBondFactor.muln(selectedCandidates.length));
+    }
+  }, [api, selectedCandidates]);
+
+  const reset = () => {
+    setAccount(undefined);
+    setAmount('');
+    setVisible(false);
+    setSelectedCandidates([]);
+  };
+
+  const onVote = () => {
+    if (module && account) {
+      startTx({
+        address: account,
+        txMethod: `${module}.vote`,
+        params: [selectedCandidates, amount],
+      });
+      reset();
+    }
+  };
+
+  return (
+    <Modal visible={visible} onDismiss={reset}>
+      <Card style={styles.modalCard}>
+        <View style={styles.centerAlign}>
+          <Text category="s1">Vote for council</Text>
+        </View>
+        <Padder scale={1} />
+
+        <Text>Vote with:</Text>
+        <Padder scale={0.5} />
+        <SelectAccount accounts={networkAccounts} selected={account} onSelect={setAccount} />
+        <Padder scale={0.5} />
+
+        <Text>Vote value:</Text>
+        <Padder scale={0.5} />
+        <Input
+          placeholder="Place your Text"
+          keyboardType="decimal-pad"
+          value={amount}
+          onFocus={() => setAmount('')}
+          onChangeText={(nextValue) => setAmount(nextValue.replace(/[^(\d+).(\d+)]/g, ''))}
+        />
+
+        <Text category="s1">{api ? formatBalance(getBalanceFromString(api, amount)) : ''}</Text>
+
+        <Padder scale={1} />
+
+        <View style={styles.centerAlign}>
+          <Text category="c1">{`Select upto ${MAX_VOTES} candidates in the preferred order:`}</Text>
```

Comment:
```suggestion
          <Text category="c1">{`Select up to ${MAX_VOTES} candidates in the preferred order:`}</Text>
```
It should be **up to**